zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -195,8 +196,24 @@ def func_test_runner(keep_model=False, smoke=False, dev=False, bundles=None,
             if ':' in bundle:
                 model_alias, bundle = bundle.split(':')
             else:
-                model_alias = utils.DEFAULT_MODEL_ALIAS
-
+                if all_bundles is None:
+                    all_bundles = {}
+                    for name, values in utils.get_charm_config().items():
+                        if '_bundles' in name:
+                            all_bundles[name] = values
+                matching_bundles = []
```

Comment:
Why not initialise `matching_bundles` to a `set()`?

Reply:
Good point, I did look at that and ended up doing this beacuse I didn't pay enough attention to the `.opo()` method on a set :)